lib/process_options.js

```diff
@@ -25,7 +26,7 @@ module.exports = function processOptions(options) {
 
       if (argName.length === 1) {
         argName = "-" + argName;
-      } else {
+      } else if(argName[0] !== "-") {
```

Comment:
It not there, overrides match `-vv` -> length > 1 -> will add `---vv`.
Or if `vv` in overrides than the result will be `--vv`, which is bad too.

Other solution is to define all options in overrides together with `-` characters and in case of overrides match, not add these characters anymore. What's better from your point of view?